requirements/pip.txt

```diff
@@ -31,8 +31,8 @@ Pygments==2.10.0
 
 # Basic tools
 redis==3.5.3
-kombu==5.2.1
-celery==5.2.0
+kombu==5.1.0
+celery==5.1.2
```

Comment:
Can you add a comment here saying how and why it breaks and ignore it by pyup for now?

Reply:
I think it's fine to try to upgrade next week, after the deploy. Didn't test if the patch version they released fixed the problem.